src/services/load_from_xml_manager.py

```diff
@@ -38,7 +38,8 @@
 RACES_DATA_PATH = "data/races.xml"
 CLASSES_DATA_PATH = "data/classes.xml"
 
-foes_by_mission: dict[str, list[Foe]] = {}
+from src.services.global_foes import foes_by_mission, link_foe_to_mission
+# foes_by_mission: dict[str, list[Foe]] = {}
```

Comment:
Please remove this commented out line of code since it's not necessary anymore

Reply:
Removed